app/backplane/sensor_module/include/c_sensor_module.h

```diff
@@ -0,0 +1,70 @@
+#ifndef C_SENSOR_MODULE_H
+#define C_SENSOR_MODULE_H
+
+#include "c_sensing_tenant.h"
+
+// F-Core Includes
+#include <f_core/c_project_configuration.h>
+#include <f_core/messaging/c_message_port.h>
+#include <f_core/os/c_task.h>
+#include <f_core/types.h>
+#include <f_core/net/application/c_udp_broadcast_tenant.h>
+
+
+class CSensorModule : public CProjectConfiguration {
```

Comment:
Can we get a simple, documented sample for CProjectConfiguration. Trying to learn the ins and outs of it might be a little daunting if you have to parse all of sensor mod (or other mods') code to figure out how its supposed to work.

Reply:
Another PR :)

Reply:
Its just a place to keep tasks and threads in one place so you can statically allocate the entire object